src/measurement-events/WoocommerceEventList.php

```diff
@@ -0,0 +1,36 @@
+<?php
+include_once 'MeasurementEventList.php';
+
+
+/**
+ * Subclass that contains information for Woocommerce plugin
+ *
+ * @class WoocommerceEventList
+ */
+class WoocommerceEventList extends MeasurementEventList {
+
+    public function __construct() {
+        $pluginName = 'Woocommerce';
+        $categories = array('ecommerce', 'ecommerce', 'ecommerce', 'ecommerce', 'ecommerce', 'ecommerce', 'ecommerce',
+            'ecommerce', 'ecommerce', 'ecommerce');
+        $actions = array('add_to_cart', 'add_to_cart', 'remove_from_cart', 'checkout', 'review_cart', 'review_cart',
+            'cart_contents', 'update_cart', 'product_details', 'place_order');
+        $selectors = array('.woocommerce-page .add_to_cart_button', '.woocommerce-page .single_add_to_cart_button',
+            '.woocommerce-page .remove', 'div.wc-proceed-to-checkout .checkout-button', 'a.added_to_cart.wc-forward',
+            'div.woocommerce-message a.wc-forward', 'a.cart-contents',
+            '.woocommerce-cart-form__contents .coupon ~ .button', '.content-area a.woocommerce-LoopProduct-link',
+            '.woocommerce-page form.woocommerce-checkout');
+        $onEvents = array('click', 'click', 'click', 'click', 'click', 'click', 'click', 'click', 'click', 'submit');
+        $secondLayerSelectors = array(null, null, null, null, 'document.body', null, null, null, null, null);
+        $secondLayerOn = array(null, null, null, null, 'added_to_cart', null, null, null, null, null);
```

Comment:
This will be hard to maintain given that the index of the element defines in which object the event is defined.

Reply:
I was thinking about this problem as well...do you think we should get this information by parsing data in a file that is grouped by event?